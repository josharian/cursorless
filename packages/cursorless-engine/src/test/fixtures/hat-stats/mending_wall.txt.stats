nTokens: 479
nHats:
	mean: 100%
	std: 0%
	min: 100%
	max: 100%
	spark: 0              25               50               75             █ 100

nPenalty0:
	mean: 8%
	std: 0%
	min: 7%
	max: 8%
	spark: 0    █         25               50               75               100

nPenalty1:
	mean: 83%
	std: 0%
	min: 82%
	max: 84%
	spark: 0              25               50               75    █          100

nPenalty2:
	mean: 10%
	std: 0%
	min: 9%
	max: 10%
	spark: 0     █▄       25               50               75               100

