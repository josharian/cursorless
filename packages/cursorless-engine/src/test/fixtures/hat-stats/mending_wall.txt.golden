*       *
Mending Wall
[-----] [--]

*  *      *    * *    **      *     *
by Robert Frost, 1914 (public domain)
[] [----] [---]# [--] #[----] [----]#

                              x      x     x
     *    *     *    *   *   **   *  *   * *
Something there is that doesn’t love a wall,
[-------] [---] [] [--] [---]## [--] # [--]#

 x                          x x          x v
 *   *       *    *  *   *  * *     *    * *
That sends the frozen-ground-swell under it,
[--] [---] [-] [----]#[----]#[---] [---] []#

v    x      v  x     x         x v   x
*    *      *  *     *         * *   *  *
And spills the upper boulders in the sun;
[-] [----] [-] [---] [------] [] [-] [-]#

w               x    v      v     v
*     *   *     *    *  *   *     *     *
And makes gaps even two can pass abreast.
[-] [---] [--] [--] [-] [-] [--] [-----]#

 w     x  x  v      v  o           x
 *     *  *  *      *  *           **
The work of hunters is another thing:
[-] [--] [] [-----] [] [-----] [---]#

w   v    x   v     o     x v      w
*   *    *   *     *     * *      *
I have come after them and made repair
# [--] [--] [---] [--] [-] [--] [----]

w            w  x     x    x v     v  # w    w
*        *   *  *     *    * *     *  * *    *
Where they have left not one stone on a stone,
[---] [--] [--] [--] [-] [-] [---] [] # [---]#

w      x     v   o  w       o  w    w      vo
*      *     *   *  *       *  *    *      **
But they would have the rabbit out of hiding,
[-] [--] [---] [--] [-] [----] [-] [] [----]#

o  o       #  v       w   x #    ^   # w   #
*  *       *  *       *   * *    *   * *   *
To please the yelping dogs. The gaps I mean,
[] [----] [-] [-----] [--]# [-] [--] # [--]#

 o   v ^    w   ^      o   x 0     0     0  ^
 *   * *    *   *      *   * *     *     *  *
No one has seen them made or heard them made,
[] [-] [-] [--] [--] [--] [] [---] [--] [--]#

o   >   #     o      v>    o  o     >      v v
*   *   *     *      **    *  *     *      * *
But at spring mending-time we find them there.
[-] [] [----] [-----]#[--] [] [--] [--] [---]#

^ v    w       #  v         # ~     w x
* *    *       *  *         * *     * *
I let my neighbor know beyond the hill;
# [-] [] [------] [--] [----] [-] [--]#

~   ^  +   o #  #     0    w  ~   0
*   *  *   * *  *     *    *  *   *
And on a day we meet to walk the line
[-] [] # [-] [] [--] [] [--] [-] [--]

  ^ o    +  ^    #       w    x   w   w
  * *    *  *    *       *    *   *   *
And set the wall between us once again.
[-] [-] [-] [--] [-----] [] [--] [---]#

0     ^       o  ^       o     >   >o
*     *  b    *  *       *  b  *   **
We keep the wall between us as we go.
[] [--] [-] [--] [-----] [] [] [] []#

+    v            w         #  #       ~   w #
*    *  b         *    g    *  *       *   * *
To each the boulders that have fallen to each.
[] [--] [-] [------] [--] [--] [----] [] [--]#

  0   ^   o     ^    v    0  #       # 0
  *   *   *     *    *    *  *       * *
And some are loaves and some so nearly balls
[-] [--] [-] [----] [-] [--] [] [----] [---]

~          #      0         o            o x
*  g    g  *   r  *    r    *   r        * *
We have to use a spell to make them balance:
[] [--] [] [-] # [---] [] [--] [--] [-----]#

x   ^    #    ^        >  +      #    o      > v
*   *    *    * p      *  *      *    *      ***
‘Stay where you are until our backs are turned!’
#[--] [---] [-] [-] [---] [-] [---] [-] [----]##

+              o                   #       >^
*    y  b      *        p   p      *       **
We wear our fingers rough with handling them.
[] [--] [-] [-----] [---] [--] [------] [--]#

  0            ^ ^     ^  0 w   0 #   >
g * *          * *     *  * *   * *   *
Oh, just another kind of out-door game,
[]# [--] [-----] [--] [] [-]#[--] [--]#

 w       ^   0    #        ^        > v
 *  r  1 *   *  y *     1  *        * *
One on a side. It comes to little more:
[-] [] # [--]# [] [---] [] [----] [--]#

            ~   0    ~  o    #           w
 y    b     *   * g  *  *    *    1   2  *
There where it is we do not need the wall:
[---] [---] [] [] [] [] [-] [--] [-] [--]#

   +   0  >      +    ~    >    ^    >
2  *   *  *      * b  *    *    *    *
He is all pine and I am apple orchard.
[] [] [-] [--] [-] # [] [---] [-----]#

 0  ~                 0   ^    0
 *  *    2     r      *   *    *
My apple trees will never get across
[] [---] [---] [--] [---] [-] [----]

    ^     0 >     >         +    ~     ~    +~
  b *     * *     *      g  *    * r   *    **
And eat the cones under his pines, I tell him.
[-] [-] [-] [---] [---] [-] [---]# # [--] [-]#

 >    > >   + w0    0        0  >         >   +o
 *    * *   * **    *        *  *         *   **
He only says, ‘Good fences make good neighbors.’
[] [--] [--]# #[--] [----] [--] [--] [-------]##

            ~    ~      #
 b     p    *    *      * b b   g y p
Spring is the mischief in me, and I wonder
[----] [] [-] [------] [] []# [-] # [----]

 >     ~       x ^           ~  +  o
 * 1   *   g   b *      2    *  *  *
If I could put a notion in his head:
[] # [---] [-] # [----] [] [-] [--]#

#  ~           >  ~         ~      + ^x x
*  *  p   b    *  *         *   *  * *b b
‘Why do they make good neighbors? Isn’t it
#[-] [] [--] [--] [--] [-------]# [-]## []

             ~  +   x  +              +  0
y       g    *  *   *  *   r     p    *  *  b   b
Where there are cows? But here there are no cows.
[---] [---] [-] [--]# [-] [--] [---] [-] [] [--]#

  ~    v    +  v      w0    ~     >
  *    b    *  b 1    b*r   *  y  *
Before I built a wall I’d ask to know
[----] # [---] # [--] ### [-] [] [--]

     o           +  ~
2    b   b       *  *  b   b      b g
What I was walling in or walling out,
[--] # [-] [-----] [] [] [-----] [-]#

            #               >   +
  p  1    g b   g    y  2   *   *     g
And to whom I was like to give offense.
[-] [] [--] # [-] [--] [] [--] [-----]#

                     w       >v      o
  r          g   r   b  y    *b g    b   r r
Something there is that doesn’t love a wall,
[-------] [---] [] [--] [---]## [--] # [--]#

 x     +   ^       ~ 0         + +       w
 b     *   b  1   r* b g       * *1    b b    pp
That wants it down.’ I could say ‘Elves’ to him,
[--] [---] [] [--]## # [---] [-] #[---]# [] [-]#

+   >                           ~ x
*   b gp b   2      *     y   2 brb r
But it’s not elves exactly, and I’d rather
[-] []## [-] [---] [-----]# [-] ### [----]

v        o              +
b  y     b b     y    p b 1     1    p
He said it for himself. I see him there
[] [--] [] [-] [-----]# # [-] [-] [---]

         #                          w
    b    b 2         r        b b   b    p
Bringing a stone grasped firmly by the top
[------] # [---] [-----] [----] [] [-] [-]

                +        o        ~
 g   r    r 1   *   p  p *   y    *     y   y
In each hand, like an old-stone savage armed.
[] [--] [--]# [--] [] [-]#[---] [----] [---]#

o    +               ^   #        x x
b    *    1    b     b   b    2   b b 2
He moves in darkness as it seems to me,
[] [---] [] [------] [] [] [---] [] []#

       x          0    #  x
2    g b        g b    b  b      r  1   1
Not of woods only and the shade of trees.
[-] [] [---] [--] [-] [-] [---] [] [---]#

^       x               v        w       x
b    y  b   g  g        b p     pb   r   b
He will not go behind his father’s saying,
[] [--] [-] [] [----] [-] [----]## [----]#

  v 0                          x  o
  b b    g     b       g     y g  b    1
And he likes having thought of it so well
[-] [] [---] [----] [-----] [] [] [] [--]

>            v
b    p   r   b yp       p     r  y         r   21
He says again, ‘Good fences make good neighbors.’
[] [--] [---]# #[--] [----] [--] [--] [-------]##

